src/walletHandlers/walletSessionStore.js

```
@@ -0,0 +1,45 @@
+        //     },
+        // }
+    }
+
+    createSession(deviceAddress, provider) {
+        const session = this.sessions.get(deviceAddress);
+        
+        if (this.sessions.has(deviceAddress)) {
+            return session;
+        } else {
+            const value = new Map([
+                ["provider", value],
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)